man/org.freedesktop.systemd1.xml

```diff
@@ -595,6 +599,8 @@ node /org/freedesktop/systemd1 {
 
     <!--property UnitsLoadFinishTimestampMonotonic is not documented!-->
 
+    <!--property UnitsLoadTimestampMonotonic is not documented!-->
```

Comment:
Not sure, but this comment is automatically added? Why only `Monotonic` version is listed here??

Reply:
I don't know either, It's been a while since I touched this PR. I could try to regenerate docs or add/remove it manually.